Bugzilla/Install/DB.pm

```diff
@@ -4287,6 +4277,13 @@ sub _add_oauth2_jwt_support {
     {TYPE => 'INT4', NOTNULL => 1});
 }
 
+sub _remove_restrict_ipaddr {
+  my $dbh = Bugzilla->dbh;
+  return unless $dbh->bz_column_info('logincookies', 'restrict_ipaddr');
+
+  $dbh->do('ALTER TABLE logincookies DROP COLUMN restrict_ipaddr');
```

Comment:
$dbh->bz_drop_column('logincookies','restrict_ipaddr');

Reply:
Ah there is! I was searching `bz_remove_column` instead 😅

Reply:
And looks like we don’t need a subroutine for that.